toqito/channel_ops/kraus_to_choi.py

```diff
@@ -61,6 +61,9 @@ def kraus_to_choi(kraus_ops: list[list[np.ndarray]], sys: int = 2) -> np.ndarray
     :return: The corresponding Choi matrix of the provided Kraus operators.
 
     """
+    if sys < 0:
```

Comment:
But this only checks the system and not the dimension, but the message is for boht system and dimension.